CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed / Improved
 
 - Adjusted the error page (#398)
+- Missing page number in pagination if total page number is even and by 1 greater than visible pages (#368)
```

Comment:
where is that change? :) I don't see changes in pagination :thinking:

Reply:
This PR https://github.com/DivanteLtd/storefront-ui/pull/1189 has been included in v0.7.12 which fixes pagination issue in Capybara.